src/Gelf/Transport/RetryTransportWrapper.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+
+namespace Gelf\Transport;
+
+use Gelf\MessageInterface as Message;
+
+
+class RetryTransportWrapper extends AbstractTransport
+{
+    /**
+     * @var AbstractTransport
+     */
+    private $transport;
```

Comment:
In order to facilitate inheritance, can you make this `protected`?

```suggestion
    protected $transport;
```

Reply:
Also, it might be interesting to add a getter function for this method to allow developers using this wrapper to access the base transport.